### ISSUE TITLE
Deeper MLP?

### ISSUE THREAD

**Issue by ildoonet** [jklj077/meProp#1]
Have you tried on deeper models?

Since each step of backprops, gradients are removed with specific portions(like 5%), Will not the gradient vanish in a deeper neural network model?

Any thoughts?

**Comment by xusun26** [comment]
Actually this method also works on deeper MLP model with 2, 3, 4, 5 hidden layers, e.g., see the Table 5 of the conference version. Thanks.
